README.md

```diff
@@ -1,151 +1,158 @@
-
-    aws-azure-login --no-sandbox
-
-### Behind corporate proxy
-
-If behind corporate proxy, then just set https\_proxy env variable.
-
-## Getting Your Tenant ID and App ID URI
-
-Your Azure AD system admin should be able to provide you with your Tenant ID and App ID URI. If you can't get it from them, you can scrape it from a login page from the myapps.microsoft.com page.
-
-1. Load the myapps.microsoft.com page.
-2. Click the chicklet for the login you want.
-3. In the window the pops open quickly copy the login.microsoftonline.com URL. (If you miss it just try again. You can also open the developer console with nagivation preservation to capture the URL.)
-4. The GUID right after login.microsoftonline.com/ is the tenant ID.
-5. Copy the SAMLRequest URL param.
-6. Paste it into a URL decoder ([like this one](https://www.samltool.com/url.php)) and decode.
-7. Paste the decoded output into the a SAML deflated and encoded XML decoder ([like this one](https://www.samltool.com/decode.php)).
-8. In the decoded XML output the value of the Issuer tag is the App ID URI.
-
-## How It Works
-
-The Azure login page uses JavaScript, which requires a real web browser. To automate this from a command line, aws-azure-login uses [Puppeteer](https://github.com/GoogleChrome/puppeteer), which automates a real Chromium browser. It loads the Azure login page behind the scenes, populates your username and password (and MFA token), parses the SAML assertion, uses the [AWS STS AssumeRoleWithSAML API](http://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRoleWithSAML.html) to get temporary credentials, and saves these in the CLI credentials file.
-
-## Troubleshooting
-
-The nature of browser automation with Puppeteer means the solution is bit brittle. A minor change on the Microsoft side could break the tool. If something isn't working, you can fall back to GUI mode (above). To debug an issue, you can run in debug mode (--mode debug) to see the GUI while aws-azure-login tries to populate it. You can also have the tool print out more detail on what it is doing to try to do in order to diagnose. aws-azure-login uses the [Node debug module](https://www.npmjs.com/package/debug) to print out debug info. Just set the DEBUG environmental variable to 'aws-azure-login'. On Linux/OS X:
-
-    DEBUG=aws-azure-login aws-azure-login
-
-On Windows:
-
-    set DEBUG=aws-azure-login
-    aws-azure-login
-
-## Support for Other Authentication Providers
-
-Obviously, this tool only supports Azure AD as an identity provider. However, there is a lot of similarity with how other logins with other providers would work (especially if they are SAML providers). If you are interested in building support for a different provider let me know. It would be great to build a more generic AWS CLI login tool with plugins for the various providers.
+[![view on npm](http://img.shields.io/npm/v/aws-azure-login.svg)](https://www.npmjs.org/package/aws-azure-login)
```

Comment:
Please make sure the diffs in this file only contains the new feature

Reply:
Oops, I didn't realize the README had CRLF line endings. But not sure how I changed them to LF. Changed back to CRLF. Maybe the .gitattributes needs to set `eol=lf` for *.md files?